app/src/org/commcare/activities/PostRequestActivity.java

```diff
@@ -110,7 +110,7 @@ private void makePostRequest() {
         if (!hasTaskLaunched && !inErrorState) {
             SimpleHttpTask postTask;
             try {
-                postTask = new SimpleHttpTask(this, url, params, true);
+                postTask = new SimpleHttpTask(this, url, params, true, null);
```

Comment:
Maybe just my own stylistic taste but I usually make an overloaded method when I find myself writing the same defaults twice as here and https://github.com/dimagi/commcare-android/pull/1627/files#diff-372dfe29c871df8f1f5baca49b82907eR182